tests/test_tools.py

```diff
@@ -104,6 +105,55 @@ def test_get_block_details_live(tools_live):
     assert "createdAt" in details
 
 
+def test_get_credits_balance(tools_mock):
+    balance = tools_mock.get_credits_balance()
+    assert isinstance(balance, dict)
+    assert "balance" in balance
+    assert True
+
+
+@pytest.mark.live
+def test_get_credits_balance_live(tools_live):
+    balance = tools_live.get_credits_balance()
+    assert isinstance(balance, dict)
+    assert "balance" in balance
+
+
+def test_get_credits_history(tools_mock):
+    credits_history = tools_mock.get_credits_history()
+    assert isinstance(credits_history, dict)
+    assert "content" in credits_history
+    assert isinstance(credits_history["content"], list)
+    assert len(credits_history["content"]) == 1
+
+
+@pytest.mark.live
+def test_get_credits_history_live(tools_live):
+    balance_history = tools_live.get_credits_history(
+        start_date="2022-03-01", end_date="2022-03-02"
+    )
+    assert isinstance(balance_history, dict)
+    assert "content" in balance_history
+    assert len(balance_history["content"]) == 32
```

Comment:
Could be that in the future this fails if test runs add additional items to the credit history of the sdk test account, maybe do `>= 32` Allthough weird that it is only 32 overall hm.

Reply:
I fixed the inputs for being just one day. Therefore, I think we should not expect changes as there could not be more transactions on this time range. But, maybe if there is some adjust on the database and they include new credits consumed on this date, then we could have an error.

Reply:
ah ja, forgot about the fixed dates.

Reply:
Checked and Done!